python/nav/jwtconf.py

```diff
@@ -8,35 +9,63 @@
 
 
 class JWTConf(NAVConfigParser):
-    """jwt.conf config parser"""
+    """webfront/jwt.conf config parser"""
 
-    DEFAULT_CONFIG_FILES = ('jwt.conf',)
+    DEFAULT_CONFIG_FILES = [join('webfront', 'jwt.conf')]
+    NAV_SECTION = "nav"
 
     def get_issuers_setting(self):
-        issuers_settings = dict()
-        for section in self.sections():
-            try:
+        """Parses the webfront/jwt.conf config file and produces a dictionary that can
```

Comment:
```suggestion
        """Parses the webfront/jwt.conf config file and returns a dictionary that can
```

Reply:
Still no type annotations, tho :(

Reply:
Added a typehint, but made it quite generic. What the dict contains is kind of all over the place

Reply:
The important part is that the return value is a dict with string keys :)